lib/std/runtime/runtime.ts

```diff
@@ -1,7 +1,59 @@
 // deno-lint-ignore-file no-explicit-any
 
 import { deepMerge } from "../../../deps.ts";
-import { REDUCE } from "./$reduce.ts";
+import { reduceNode } from "./node.ts";
+
+function resolveChildren<TResult, TValue>(
+  initial: TResult,
+  root: TValue,
+) {
+  // I don't get it...
+  // console.log({ result, children });
```

Comment:
TODO: Remove all unused code and comments.

Reply:
Resolved in <https://github.com/FartLabs/jsonx/pull/4/commits/be3375e1906ababf00822f3bd04f48bdd6d86682>.